mos-platform/nes-mmc1/prg-rom-banked.ld

```diff
@@ -88,7 +152,7 @@ SECTIONS {
   .reset_fixed : { KEEP(*(.reset_fixed)) } >reset_fixed
 }
 
-PROVIDE(__prg_rom_size = 256);
+PROVIDE(__prg_rom_size = 512);
```

Comment:
We should still default to 256 I think. We've tended to set our defaults to the most likely configuration, not the most capable. (Although among the most likely configurations, we prefer the most capable.)